### PR TITLE
[zh-cn]sync cpu-management-policies.md pull-image-private-registry.md

### PR DESCRIPTION
**Pull request by my-git9**
[zh-cn]sync cpu-management-policies.md pull-image-private-registry.md

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/cpu-management-policies.md [hunk 1]
```diff
@@ -479,7 +479,7 @@ than number of NUMA nodes.
 注意，此策略选项不兼容 `TopologyManager` 与 `single-numa-node` 策略，并且不适用于 CPU 的插槽数量大于 NUMA 节点数量的硬件。
 
 <!--
-The `full-pcpus-only` option can be enabled by adding `full-pcups-only=true` to
+The `full-pcpus-only` option can be enabled by adding `full-pcpus-only=true` to
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/cpu-management-policies.md:482]: Does the zh text need to changed too? Line 493.

**Review by windsonsea — COMMENTED** [review]
sync the zh text too pls.

**Comment by tengqm** [comment]
/approve

Please recommit to trigger the CI again.

**Comment by Sea-n** [comment]
Hey @my-git9,

Please re-trigger CI one more time, thanks.

**Pull request merged**